bin/helm-generate-raw-manifests-in-docker

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -e
+
+DIFF_ONLY=false
+readonly REPO_ROOT="$(git rev-parse --show-toplevel)"
+readonly HELM_CHARTS=("conjur-config-cluster-prep" "conjur-config-namespace-prep")
+
+function main() {
+  while true; do
+    case "$1" in
+      -d|--diff-only )
+        DIFF_ONLY=true
+        shift
+        ;;
+      * )
+        if [ -z "$1" ]; then
+            break
+        else
+            echo "$1 is not a valid option"
+            exit 1
+        fi
+        ;;
+    esac
+  done
+
+  # Backup old manifests
+  for chart in "${HELM_CHARTS[@]}"; do
+    mv "helm/$chart/generated/$chart.yaml" "helm/$chart/generated/$chart.old.yaml"
+  done
+  
+  echo "Building utility image with helm and yq..."
+  docker build -f Dockerfile.helm-unit-test -t conjur-k8s-helm-generate-manifests:dev .
+
+  echo "Generating raw k8s manifests..."
+  docker run --rm \
+    --name conjur-k8s-helm-generate-manifests \
+    -v "$REPO_ROOT:/conjur-authn-k8s-client" \
+    --entrypoint "/conjur-authn-k8s-client/bin/helm-generate-raw-manifests" \
+    conjur-k8s-helm-generate-manifests:dev
+
+  manifests_changed=false
+  for chart in "${HELM_CHARTS[@]}"; do
+    echo -e "\nDiffing generated/$chart.yaml..."
+    set +e
```

Comment:
It's possible to do this without temporarily setting `set +e`:
```
      ret_val=0
      diff -cs helm/$chart/generated/"$chart.old.yaml" helm/$chart/generated/"$chart.yaml" || ret_val=$?
```

Reply:
Good to know, thanks for the tip! Changed